packages/telestion-client-common/src/components/pages/login-page/login-page.tsx

```diff
@@ -1,8 +1,66 @@
+import { ReactElement } from 'react';
 import { Flex } from '@adobe/react-spectrum';
 import { Routing } from '@wuespace/telestion-client-types';
-import { Login } from './login';
 
-export function LoginPage() {
+/**
+ * React Props of {@link LoginPageProps}
+ *
+ * For more information about React Props, please look here:
+ * {@link https://reactjs.org/docs/components-and-props.html}
+ *
+ * @see {@link LoginPageProps}
+ * @see {@link https://reactjs.org/docs/components-and-props.html}
+ */
+export interface LoginPageProps {
+	/**
+	 * Components that the login page should render.
+	 *
+	 * Typical pre-defined components are:
+	 * - {@link LoginTitle}
+	 * - {@link LoginLogo}
+	 * - {@link LoginDescription}
+	 * - {@link LoginForm}
+	 */
+	children: ReactElement | Array<ReactElement>;
+}
+
+/**
+ * A Telestion Client page that renders a login page
+ * where the user typically enters his credentials
+ * to login in the application.
+ *
+ * It only renders if nobody is authenticated
+ * otherwise it redirects to the dashboard page.
+ *
+ * It defines vertical display layout to align its children.
```

Comment:
```suggestion
 * A Telestion Client page that renders a login page
 * where the user typically enters their credentials
 * to log into the application.
 *
 * It only renders if nobody is authenticated.
 * Otherwise, it redirects to the dashboard page.
 *
 * It aligns its children vertically.
```